tests/approaches/test_oracle_approach.py

```diff
@@ -137,12 +139,17 @@ def _policy_solves_task(policy, task, simulator):
 def test_oracle_approach(env_name, env_cls):
     """Tests for OracleApproach class with all environments."""
     for extra_args in EXTRA_ARGS_ORACLE_APPROACH[env_name]:
-        utils.reset_config({
+        args = {
```

Comment:
would it be cleaner to do this by doing:
```
args_dict = {
            "env": env_name,
            "num_train_tasks": 2,
            "num_test_tasks": 2,
}
args_dict.update(extra_args)  # note: extra_args can override the default values in args_dict
utils.reset_config(args_dict)
```
? is this less clear than what you have?

Reply:
I think mine is slightly more clear but I don't feel strongly

Reply:
I'm just gonna leave it